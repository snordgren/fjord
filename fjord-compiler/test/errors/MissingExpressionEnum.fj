module test.errors.MissingExpressionEnum

enum A
  B : A
  C : A

missing : Int
missing = 
 