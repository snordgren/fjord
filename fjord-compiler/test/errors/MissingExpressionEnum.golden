test/errors/MissingExpressionEnum.fj:9:2:
  |
9 |  
  |  ^
unexpected end of input
expecting expression
